py/desitarget/sv2/sv2_cuts.py

```diff
@@ -277,8 +277,8 @@ def isBACKUP(ra=None, dec=None, gaiagmag=None, primary=None):
 
 def isLRG(gflux=None, rflux=None, zflux=None, w1flux=None, w2flux=None,
           zfiberflux=None, rfluxivar=None, zfluxivar=None, w1fluxivar=None,
-          gnobs=None, rnobs=None, znobs=None, maskbits=None, primary=None,
-          south=True):
+          gaiagmag=None, gnobs=None, rnobs=None, znobs=None, maskbits=None,
```

Comment:
Here, you have correctly added `gaiagmag=None`. But, you also need to pass `gaiagmag` to the `isLRG()` function in the first place. 

Down around line 1860 you'll see this code:
```
     if "LRG" in tcnames:
         for south in south_cuts:
             lrg_classes[int(south)] = isLRG(
```
you need to also make sure to pass `gaiagmag=gaiagmag` to the `isLRG` function at that point in the code.